gemfirexd/core/src/drda/java/com/pivotal/gemfirexd/internal/impl/drda/ClientThread.java

```diff
@@ -104,6 +104,7 @@ public Object run() throws IOException
                         else {
                           clientSocket.setKeepAlive(false);
                         }
+                        clientSocket.setTcpNoDelay(true);
```

Comment:
Just for my information, this will not have much impact when result set is large because we will anyway be writing large data at a time?